src/utils/fetchClient.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Consider removing 'eslint-disable @typescript-eslint/no-explicit-any' and replacing 'any' type with more specific types where possible.